MedTracker/Controllers/DoctorController.cs

```diff
@@ -154,33 +153,68 @@ public async Task<IActionResult> GetPatients()
                     p => p.IdUser,
                     (tm, p) => new
                     {
-                        tm.IdTreatment,
                         p.IdUser,
                         p.FirstName,
                         p.LastName,
                         p.PhoneNumber
                     })
                 .ToListAsync();
 
+            return Ok(patients);
+        }
+        catch (Exception ex)
+        {
+            return StatusCode(500, $"An error occurred: {ex.Message}");
+        }
+    }
 
-            var patientsWithTreatments = new List<object>();
+    [HttpGet("GetPatientTreatments")]
```

Comment:
We need to receive the id for the user whose treatments we are interested. Just look at the name: is `GetPatient_Treatments`, not `GetTreatments`.  There are some methods which receive just one id as input, use them as reference for this method.

Reply:
We need receive all patients and all their treatments for the list of treatments in front. And after this if we want to see another patient info i can do another method but menthors dont mentionated it. We need only name and phone number.

Reply:
this is not ok from api perspective. you mean we should return bulk data every time we need some data? To have method GetPatients(), GetPatientsWithTreatments(), GetPatientsAndTeirDoctors(), GetTreatmentsAndDoctorsDataAndPatientData()?

Reply:
But ok, **just** for the demo, I'll merge this